.github/workflows/release.yaml

```diff
@@ -319,7 +320,30 @@ jobs:
           prerelease: false
           draft: true
           fail_on_unmatched_files: true
-
+      - name: Update index helm file
+        uses: actions/checkout@v3
+        run: |
+          echo "Updating Helm repo index"
+          ./hack/build/ci/generate-new-helm-index-yaml.sh
+      - name: Create pull request for adding helm index to main branch
+        uses: peter-evans/create-pull-request@v5
```

Comment:
is this action the way to go for creating PRs? looks like it comes out of a private repo

Reply:
Repo is public https://github.com/peter-evans/create-pull-request